### DISCUSSION TITLE
app is safe? asking me to log in via dedicated in-app sign in page

### DISCUSSION THREAD

**Discussion by hdgk12jh3kj12h3lk** [z-huang/music#1656] [General]
thank you!!!

**Comment by z-huang** [comment]
Yes, it's safe. This app is open-sourced. Everyone can get the code and check it.

**Reply by hdgk12jh3kj12h3lk** [reply]
thank you friend! I appreciate your hard work. Problem is - no one does :) but that's a general truth (not specific to your app). i noticed it won't play unless signed in. i was wondering if it's perhaps possible to play without a sign in. another option is to do what NewPipe does and ask the user to Google Hangout his YTMusic. T**hank you** again!!!!
